index.js

```diff
@@ -332,14 +327,16 @@ module.exports = function (babel) {
       let scope = shouldUseAutomaticScope(options) ? getScope(path.scope) : null;
       let strict = shouldUseStrictMode(options);
 
-      let emberIdentifier = state.ensureImport('default', 'ember');
+      let emberIdentifier = state.ensureImport('createTemplateFactory', '@ember/template-factory');
 
       replacePath(
         path,
         state,
         compileTemplate(precompile, template, emberIdentifier, { isProduction, scope, strict }),
         options
       );
+
+      path.scope.crawl();
```

Comment:
Why do we have to unconditionally recrawl? If it is due to the new import, we should add the recrawl to `ensureImport`?

Reply:
We need to both add the import to bindings, and _also_ add the _usages_ to bindings. Otherwise, `babel-plugin-ember-modules-api-polyfill` can't rename the usages either. The recrawl gets the usages, not the import.

I actually want to add a test for this to make sure it works with the other paths as well, my assumption was that `crawl` only re-crawled that path, but if it does the whole file that's definitely problematic.